pkgpanda/__init__.py

```diff
@@ -356,7 +356,7 @@ def load(self, id: str):
         filename = os.path.join(path, "pkginfo.json")
         try:
             pkginfo = load_json(filename)
-        except FileNotFoundError as ex:
+        except OSError as ex:
```

Comment:
> We aren't looking for a specific error here

I meant, in the generic sense. For e.g, one could argue that using `except Exception`  is perfectly alright here because we are going to raise `PackageError` immediately afterward.   But, in general, it is agreed upon by many that `except Exception` is a bad idea and looking for focussed exception is better.  Some folks think that the type of Exception being caught helps while reading or understanding the code.  

> we aren't certain what the cause is until runtime.

Please note that there are multiple places of this load_json and exception in the scenario. So, that needs to be kept in mind, (Like line 779 and line 519 `build/__init__.py`  (This is an important point to consider because we introduce an inconsistency here).

On 2. I meant "earlier" that this stanza, like ensuring that path + id is definitely a path.  I agree that printing the error message gives better error info on the exact reason for the failure.

If the inconsistency point is address, I am fine with this change.